kernel/kernel-api/src/main/java/io/delta/kernel/transaction/UpdateTableTransactionBuilder.java

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright (2025) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.kernel.transaction;
+
+import io.delta.kernel.Transaction;
+import io.delta.kernel.annotation.Evolving;
+import io.delta.kernel.engine.Engine;
+import io.delta.kernel.exceptions.*;
+import io.delta.kernel.expressions.Column;
+import io.delta.kernel.types.StructType;
+import io.delta.kernel.utils.CloseableIterable;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * Builder for creating a {@link Transaction} to update an existing Delta table.
```

Comment:
instead of seeing the word `creating` here (which could be a tiny bit confusing, since this is an UPDATE txn builder not a CREATE txn builder` --> could we just say `Builds a {@link Transaction} to update an existing ..`